content/articles/create-screen-recorder-in-javascript/index.md

```diff
@@ -0,0 +1,219 @@
+title: Section Engineering Education
+
+description: Resources created by engineers for engineers
+
+aliases:
+  - '/create-screen-recoder/'
+  - '/javascript-screen-recorder/'
+type: articles
+
+images:
+  - url: /engineering-education/create-screen-recorder-in-javascript/hero.jpg
+  
+    alt: Screen Recorder in JavaScript
+
+
+### Introduction
+The Screen Capture API is used to capture the screen or a part of the screen as a Media Stream. From the recorded Media Stream we can create a video file.
```

Comment:
This is a good attempt but I still think you can make the introduction a little more detailed. Give us more background details.

Reply:
i have changed this